src/utils/bluetoothPermissions.ts

```diff
@@ -14,8 +14,8 @@ import {
 /**
  * Shows an alert if device's bluetooth is powered off
  */
-export const showBluetoothPoweredOffAlert = async () => {
-  await Alert.alert(i18n.t(i18n.l.bluetooth.powered_off_alert.title), i18n.t(i18n.l.bluetooth.powered_off_alert.message), [
+export const showBluetoothPoweredOffAlert = () => {
+  Alert.alert(i18n.t(i18n.l.bluetooth.powered_off_alert.title), i18n.t(i18n.l.bluetooth.powered_off_alert.message), [
```

Comment:
can you verify that await is not needed here and for the one beneath this as well?